Appimage for the new version (9.0) doesn't work in ubuntu 18.04 when the previous version worked.

```
n6401@mi:~$ '/home/n6401/bin/nteract-0.9.0-x86_64.AppImage' 
module.js:545
    throw err;
    ^

Error: Cannot find module '/tmp/.mount_nterac6StaSc/resources/electron.asar/browser/init.js'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:473:25)
    at Function.Module.runMain (module.js:683:10)
    at startup (bootstrap_node.js:196:16)
    at bootstrap_node.js:622:3
```

--- a/applications/desktop/package.json
+++ b/applications/desktop/package.json
@@ -129,7 +129,7 @@
     "cross-env": "^5.1.3",
     "date-fns": "^1.29.0",
     "electron": "2.0.2",
-    "electron-builder": "^20.0.0",
+    "electron-builder": "20.14.7",
     "electron-context-menu": "^0.9.1",
     "electron-log": "^2.2.13",
     "electron-updater": "^2.18.0",

